Android build issue

When trying to build the project on Android this issue prevents me from running the project:
```
/Users/alexandermann/Code/Whisk/whisk-mobile-client/node_modules/react-native-awesome-card-io/android/src/main/java/com/cardio/RNCardIOPackage.java:23: error: method does not override or implement a method from a supertype
    @Override
    ^
1 error
:react-native-awesome-card-io:compileReleaseJavaWithJavac FAILED

FAILURE: Build failed with an exception.
```
The only part in my project that depends on the CardIO module is `react-native-checkout`

--- a/package.json
+++ b/package.json
@@ -14,14 +14,14 @@
   ],
   "private": false,
   "main": "src/index.js",
-  "repository" : {
-    "type" : "git",
-    "url" : "https://github.com/z-dev/react-native-checkout"
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/z-dev/react-native-checkout"
   },
   "dependencies": {
     "lodash": "^4.14.2",
     "payment": "^2.2.1",
-    "react-native-awesome-card-io": "^0.6.1",
+    "react-native-awesome-card-io": "0.7.0",
     "react-native-keyboard-spacer": "^0.3.0",
     "string": "^3.3.3"
   },

